crates/api_models/Cargo.toml

```diff
@@ -11,14 +11,12 @@ errors = [
     "dep:reqwest",
 ]
 multiple_mca = []
-dummy_connector = []
+dummy_connector = ["common_enums/dummy_connector"]
 detailed_errors = []
 
 [dependencies]
 actix-web = { version = "4.3.1", optional = true }
 error-stack = "0.3.1"
-frunk = "0.4.1"
-frunk_core = "0.4.1"
```

Comment:
add frunk back. Since this will be a major refactor in other crates as they are still dependent on the `ForeignFrom` conversions. You can take up the refactoring on the dependent crates slowly. Create an issue stating to remove the `frunk` and `ForeignFrom` dependencies.